classes/Conf/class.xoctConfFormGUI.php

```diff
@@ -159,6 +166,11 @@ public static function checkItem($item)
      */
     protected function initAPISection()
     {
+        global $DIC;
```

Comment:
@ferishili could you move this (as $this->main_template = ...) to the constructor?